Resources/Private/Fluid/Partials/NewsItem.html

```diff
@@ -5,15 +5,33 @@
 <div class="news-list-item">
     <span class="news-list-date">{ma.NEWS_DATE}</span>
     <h2>
-        <f:link.typolink parameter="{conf.singlePid}" additionalParams="&tx_ttnews[tt_news]={news.row.uid}">
-            {ma.NEWS_TITLE}
-        </f:link.typolink>
+        <f:if condition="{news.row.type} == 2">
+            <f:then>
+                <f:link.typolink parameter="{news.row.ext_url}">
+                    {ma.NEWS_TITLE}
+                </f:link.typolink>
+            </f:then>
+            <f:else>
+                <f:link.typolink parameter="{conf.singlePid}" additionalParams="&tx_ttnews[tt_news]={news.row.uid}">
+                    {ma.NEWS_TITLE}
+                </f:link.typolink>
+            </f:else>
+        </f:if>
     </h2>
 
     <f:if condition="{news.images.0}">
-        <f:link.typolink parameter="{conf.singlePid}" additionalParams="&tx_ttnews[tt_news]={news.row.uid}">
-            <f:image image="{news.images.0}" maxWidth="{conf.displayList.image.file.maxW}" minHeight="{conf.displayList.image.file.maxH}" />
-        </f:link.typolink>
+
+        <f:if condition="{news.row.type} == 2">
+            <f:then>
+                <f:link.typolink parameter="{news.row.ext_url}">
+                    <f:image image="{news.images.0}" maxWidth="{conf.displayList.image.file.maxW}" maxHeight="{conf.displayList.image.file.maxH}" />
+                </f:link.typolink>
+            </f:then>
+            <f:else>
+                <f:link.typolink parameter="{conf.singlePid}" additionalParams="&tx_ttnews[tt_news]={news.row.uid}">
+                    <f:image image="{news.images.0}" maxWidth="{conf.displayList.image.file.maxW}" maxHeight="{conf.displayList.image.file.maxH}" />
+                </f:link.typolink>
+            </f:else>
```

Comment:
fluid parse error in template partial `NewsItem` due to missing closing `</f:if>`